Mathlib/Data/Bool/AllAny.lean

```diff
@@ -0,0 +1,85 @@
+/-
+Copyright (c) 2017 Mario Carneiro. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Mario Carneiro
+
+! This file was ported from Lean 3 source module data.bool.all_any
+! leanprover-community/mathlib commit 5a3e819569b0f12cbec59d740a2613018e7b8eec
+! Please do not edit these lines, except to modify the commit id
+! if you have ported upstream changes.
+-/
+import Mathlib.Data.List.Basic
+
+/-!
+# Boolean quantifiers
+
+This proves a few properties about `list.all` and `list.any`, which are the `bool` universal and
+existential quantifiers. Their definitions are in core Lean.
+-/
+
+
+variable {α : Type _} {p : α → Prop} [DecidablePred p] {l : List α} {a : α}
+
+namespace List
+
+--#print List.all_nil
+/-
+@[simp]
+theorem all_nil (p : α → Bool) : all [] p = tt :=
+  rfl
+#align list.all_nil List.all_nil
+-/
+
+/- warning: list.all_cons -> List.all_cons is a dubious translation:
+lean 3 declaration is
+  forall {α : Type.{u1}} (p : α -> Bool) (a : α) (l : List.{u1} α), Eq.{1} Bool (List.all.{u1} α (List.cons.{u1} α a l) p) (and (p a) (List.all.{u1} α l p))
+but is expected to have type
+  forall {α : Type.{u1}} {p : α} {a : List.{u1} α} {l : α -> Bool}, Eq.{1} Bool (List.all.{u1} α (List.cons.{u1} α p a) l) (and (l p) (List.all.{u1} α a l))
+Case conversion may be inaccurate. Consider using '#align list.all_cons List.all_consₓ'. -/
+@[simp]
+theorem all_consₓ (p : α → Bool) (a : α) (l : List α) : all (a :: l) p = (p a && all l p) :=
```

Comment:
`all_cons` and `any_cons` are also in the `lake-packages/std/Std/Data/List/Init/Lemmas.lean` and I followed the suggested to add ₓ.

Reply:
I think you're only supposed to add the  ₓ to the align statement and not the actual lemma name

Reply:
got it, done

Reply:
but then it doesn't build...

Reply:
done